data/transactions/aggregates_test.go

```diff
@@ -57,3 +57,17 @@ func TestPaysetDoesNotCommitToSignatures(t *testing.T) {
 	commit2 := payset.Commit(false)
 	require.Equal(t, commit1, commit2)
 }
+
+func TestEmptyPaysetCommitment(t *testing.T) {
```

Comment:
The test is good.. but does not cover the non-flat Commits. I know that we stopped supporting this by version 10, but if we're to retain the support and add a unit test to validate that, than we should validate both cases.
I would not feel bad if we were to drop the support for non-flat Commits. ( since its not in use )

Reply:
added!